src/Event/Event.styles.js

```diff
@@ -1,13 +1,17 @@
 import { StyleSheet } from 'react-native';
 
 const styles = StyleSheet.create({
-  item: {
+  container: {
     alignItems: 'center',
     position: 'absolute',
     borderRadius: 0,
     flex: 1,
     overflow: 'hidden',
   },
+  touchableContainer: {
+    flex: 1,
+    alignSelf: 'stretch',
```

Comment:
Could I know the reason we use `alignSelf`?

Reply:
**If I remember correctly**, `flex: 1` stretches the `TouchableOpacity` in one direction (e.g. the vertical direction, and `alignSelf: 'stretch'` stretches the `TouchableOpacity` in the other direction (horizontally). That way the whole event-box is pressable.